test/parallel/test-npm-install.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 const common = require('../common');
+if (!common.hasCrypto) {
+  common.skip('missing crypto');
+  return;
+}
```

Comment:
@mscdex Your assumption is correct; npm uses request which `require()`s `https` during its startup phase, it doesn’t even have to talk to the registry for this to fail.